package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "native-template",
-  "version": "7.0.3",
+  "version": "7.0.4-beta.1",
```

Comment:
Why are we releasing this as beta version?

Reply:
Based on our discussion, I'll prepare a pre-release without the 'beta' label in the version name.